### ISSUE TITLE
Implementing forms

### ISSUE THREAD

**Issue by AlexKhymenko** [Caballerog/ng2-inline-editor#6]
Hi im new to angular2 are you planing of implementing x-editable forms.If not can you give me some hints what is the best way to do it in angular2

**Comment by Caballerog** [comment]
Yes, I've in mind implement x-editable forms but I'm not know when I could do it. Although probably I must create a new component to do this.

You could start reading about custom forms controls in angular2:  http://blog.thoughtram.io/angular/2016/07/27/custom-form-controls-in-angular-2.html
